### PR TITLE
safely paste from word 

### PR DESCRIPTION
**Pull request by eritiro**
javascript that intercept the paste event on textareas and removes word-specific characters in order to safely import text from word

### CODE REVIEW

**Comment by eritiro** [comment]
Since this is a kind of manually "replace", It would be nice to test it with a client example.

**Comment by eritiro** [comment]
https://trello.com/c/a83j5Nhw/766-qae2015-strip-ms-word-specific-characters

**Comment by rusllonrails** [comment]
Looks reasonably well :+1:

File: app/assets/javascripts/clean-paste.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+$(function(){
+  $("textarea").bind('paste',function(e){
+    var rte = $(this);
+    setTimeout(function(){
+        var text = rte.val();
+        text = text.replace(/(.*(?:endif-->))|([ ]?<[^>]*>[ ]?)|(&nbsp;)|([^}]*})/g,'');
```
Comment (matthewford) [inline app/assets/javascripts/clean-paste.js]: where is this list from. I think there are more of these symbols.

**Pull request merged**